Tools/python/CondorSubmitter.py

```diff
@@ -440,15 +449,16 @@ def _submit(cls, working_dir, njobs):
         cwd = os.getcwd()
         os.chdir(working_dir)
         try:
-            submit_out, submit_ret = popen('condor_submit < cs_submit.jdl', return_exit_code=True)
+            #submit_out, submit_ret = popen('condor_submit cs_submit.jdl', return_exit_code=True) #Alec changed 'condor_submit < cs_submit.jdl' to how it appears now
+            submit_out, submit_ret = popen('ssh `uname -n` "export X509_USER_PROXY=%s; cd %s; condor_submit < cs_submit.jdl"' % (os.environ['X509_USER_PROXY'], working_dir), return_exit_code=True)
             ok = False
```

Comment:
We're doing everything based on https://www.evernote.com/shard/s376/nl/66335180/5f87d8cb-1733-5633-57c1-8546ed2b9cde these days, right? The "ssh `uname -n`" trick was a temporary thing, before the more modern solution came into play. We should revert it, unless there's some reason that it no longer works.

Reply:
(probably a comment for @duquale, but @pkotamnives and @abbywarden should chime in too, if they rely on this)